com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/cf/clients/ServiceWithAlternativesCreatorTest.java

```diff
@@ -132,12 +135,13 @@ public void setUp() {
     }
 
     private void setUpServiceRequests() {
+        Mockito.reset(getMockedRestTemplate());
```

Comment:
Why?

Reply:
We reset the mocks for every case because there is a chanse that the mocked behavior can be passed to the next test case where it will not be overriden for example if we need to throw an exception. Look at the code below the reset.